search-app/resources/templates/sitemap-provider-tag.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+ {% for link in links %}
+  <url>
+    <loc>{{ link.href }}</loc>
+    <lastmod>{{ datestamp }}</lastmod>
```

Comment:
I missed the tag in that comment. 😄 

<changefreq>

But after reading through everything it looks like we're regenerating on every request which would imply "always". I don't think we'd want anything to crawl more than once a day though, so maybe we should just specify "daily". Do you think it makes sense including and would "daily" be ok?